src/ts/contentScripts/duplicate-block-content/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { browser } from 'webextension-polyfill-ts';
+import { getActiveEditElement } from '../../utils/dom';
+
+const pasteEvent = new Event('input', {
+    bubbles: true,
+    cancelable: true,
+});
+const inputEvent = new Event('input', {
+    bubbles: true,
+    cancelable: true,
+});
+
+const duplicate = () => {
+    const element = getActiveEditElement() as HTMLTextAreaElement;
```

Comment:
we should probably change the getActiveEditElement to check if it's text area and return it or null, as we only want textareas anyhow :p

Reply:
btw this file somehow got in when I rebased my branch into master - not sure what happened

Reply:
hmm, that's weird it should only be present in the duplication branch